faithcomesbyhearing/dbp
# FCBH-955 Ability to change font of mobile and web

# Description
- Added fonts, bible_filesets_fonts migration
- Added Models in order to show the fonts on the bible object

## Issue Link
Original Story: [FCBH-955](https://fullstacklabs.atlassian.net/browse/FCBH-955) 

## How Do I QA This
- On your local environment run `php artisan migrate --database="dbp_users"` to run the migrations
- Insert a font entry to `fonts` table. 
- Assign the new font to the `ENGESV` text fileset on the `bible_fileset_fonts` table
- Test `https://dbp.test/api/bibles/ENGESV?asset_id=dbp-prod&v=4&key={API_KEY}` GET and verify that you get the fonts

**(*)Note:** If you want to reverse the migration run `php artisan migrate:rollback --database="dbp_users"`

## Checklist
